clang-tools-extra/unittests/clang-tidy/PerformanceModuleTest.cpp

```diff
@@ -0,0 +1,57 @@
+#include "ClangTidyTest.h"
```

Comment:
Is there a point to these unit tests if they're already in the functional tests?

Reply:
I don't think so, they were added early on. I am of the belief that they don't really solve any purpose here.